app/src/main/java/com/gh4a/resolver/LinkClickHandler.java

```diff
@@ -0,0 +1,33 @@
+package com.gh4a.resolver;
+
+import android.net.Uri;
+
+import com.gh4a.BaseActivity;
+import com.gh4a.utils.IntentUtils;
+
+public class LinkClickHandler {
+    private final BaseActivity mActivity;
+
+    public LinkClickHandler(BaseActivity mActivity) {
+        this.mActivity = mActivity;
+    }
+
+    public void handleClick(Uri clickedUri) {
```

Comment:
Since this class does not have any internal state, this method can/should very well be `public static void handleClick(BaseActivity activity, Uri clickedUri)` ... and maybe be moved to LinkSpan, just because classes with a single static method IMHO just add clutter without providing too much benefit.

Reply:
Yes, makes sense. Even though calling `LinkSpan.handleClick` would sound a bit confusing when you aren't dealing with actual link spans, at least to me. That's why I extracted the class in the first place.